oem/ibm/libpldmresponder/utils.cpp

```diff
@@ -230,15 +259,43 @@ void clearLicenseStatus()
 
     auto data = convertBinFileToJson(curLicFilePath);
 
-    const Json empty{};
+    // Guard: check that we got an object and not null/array/primitive
+    if (!data.is_object())
+    {
+        return;
+    }
+    const Json emptyJson{};
     const std::vector<Json> emptyList{};
-
-    auto entries = data.value("Licenses", emptyList);
+    std::vector<Json> entries;
+    try
+    {
+        entries = data.value("Licenses", emptyList);
+    }
+    catch (const std::exception& e)
+    {
+        error(
```

Comment:
same here - https://github.com/openbmc/docs/blob/master/anti-patterns.md#ill-defined-data-structuring-in-lg2-message-strings